profiles/serializers.py

```diff
@@ -33,7 +33,7 @@ def create(self, validated_data):
             user = get_user_model().objects.create(**validated_data)
             Profile.objects.create(user=user, **profile_data)
 
-        get_or_create_user(user.username)
+        get_auth_tokens(user)
```

Comment:
Should we call `_configure_access_token` here too? Is there any case where we would want to call `get_auth_token` and not also `_configure_access_token`?

Reply:
No, `_confiure_access_token` only configured a reddit client, which we don't need here. `get_auth_tokens` is the new equivalent of `get_or_create_user`